### PR TITLE
Paintroid-158 Add share image button

### PR DESCRIPTION
**Pull request by Electronix1337**
Added share image button to the overflow menu.

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by geolas94 — APPROVED** [review]
lgtm, test failure from jenkins

File: Paintroid/src/main/java/org/catrobat/paintroid/ui/MainActivityNavigator.java [hunk 1]
```diff
@@ -135,6 +137,19 @@ public void startWelcomeActivity(@ActivityRequestCode int requestCode) {
...
+	@Override
+	public void startShareImageActivity(Bitmap bitmap) {
+		Uri uri = FileIO.saveBitmapToCache(bitmap, mainActivity);
+		if (uri != null) {
+			Intent shareIntent = new Intent();
+			shareIntent.putExtra(Intent.EXTRA_STREAM, uri);
+			shareIntent.setDataAndType(uri, mainActivity.getContentResolver().getType(uri));
+			shareIntent.setFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
+			shareIntent.setAction(Intent.ACTION_SEND);
+			mainActivity.startActivity(Intent.createChooser(shareIntent, "Send image via"));
```
Comment (StanglTobias) [inline Paintroid/src/main/java/org/catrobat/paintroid/ui/MainActivityNavigator.java:149]: don't use hardcoded strings that need to be translated. extract **"send image via"** to string resource

**Review by StanglTobias — CHANGES_REQUESTED** [review]

**Review by StanglTobias — APPROVED** [review]

**Review by wslany — CHANGES_REQUESTED** [review]
Crashes with 
10-17 18:10:00.769 E/AndroidRuntime(20033): FATAL EXCEPTION: main
10-17 18:10:00.769 E/AndroidRuntime(20033): Process: org.catrobat.paintroid.independent_20201009_2251, PID: 20033
10-17 18:10:00.769 E/AndroidRuntime(20033): java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.res.XmlResourceParser android.content.pm.ProviderInfo.loadXmlMetaData(android.content.pm.PackageManager, java.lang.String)' on a null object reference
10-17 18:10:00.769 E/AndroidRuntime(20033): 	at androidx.core.content.FileProvider.parsePathStrategy(FileProvider.java:605)
10-17 18:10:00.769 E/AndroidRuntime(20033): 	at androidx.core.content.FileProvider.getPathStrategy(FileProvider.java:579)
10-17 18:10:00.769 E/AndroidRuntime(20033): 	at androidx.core.content.FileProvider.getUriForFile(FileProvider.java:417)
10-17 18:10:00.769 E/AndroidRuntime(20033): 	at org.catrobat.paintroid.FileIO.saveBitmapToCache(FileIO.java:100)
10-17 18:10:00.769 E/AndroidRuntime(20033): 	at org.catrobat.paintroid.ui.MainActivityNavigator.startShareImageActivity(MainActivityNavigator.java:142)
10-17 18:10:00.769 E/AndroidRuntime(20033): 	at org.catrobat.paintroid.presenter.MainActivityPresenter.shareImageClicked(MainActivityPresenter.java:198)
10-17 18:10:00.769 E/AndroidRuntime(20033): 	at org.catrobat.paintroid.MainActivity.onOptionsItemSelected(MainActivity.java:225)
10-17 18:10:00.769 E/AndroidRuntime(20033): 	at android.app.Activity.onMenuItemSelected(Activity.java:3544)

**Review by wslany — APPROVED** [review]

**Comment by wslany** [comment]
Thanks!

**Pull request merged**